zoo-css-challenge/index.html

```diff
@@ -78,8 +78,8 @@ <h3>Height and Size</h3>
 			<p>Giraffes use their height to good advantage and browse on leaves and buds in treetops that few other animals can reach (acacias are a favorite). Even the giraffe's tongue is long! The 21-inch tongue helps them pluck tasty morsels from branches. Giraffes eat most of the time and, like cows, regurgitate food and chew it as cud. A giraffe eats hundreds of pounds of leaves each week and must travel miles to find enough food.</p>
 		</section>
 
-		<section class="news">
-			<h2>News</h2>
+		<h2 class="news">News</h2>
+		<section class="news" id="card-flex">
```

Comment:
```suggestion
		<section class="news" id="card-flex">
			<h2 class="news">News</h2>
```
It looks like these two elements were swapped around — we should change them back as none of the tasks were to take the `h2` out of the `section`